launcher/modplatform/flame/FlameAPI.h

```diff
@@ -1,35 +1,61 @@
 #pragma once
 
 #include "modplatform/helpers/NetworkModAPI.h"
+#include "Application.h"
 
 class FlameAPI : public NetworkModAPI {
+   private:
+    inline auto getSortFieldInt(QString sortString) const -> int {
+        return sortString == "Featured"
+          ? 1
+          : sortString == "Popularity"
+          ? 2
+          : sortString == "LastUpdated"
+          ? 3
+          : sortString == "Name"
+          ? 4
+          : sortString == "Author"
+          ? 5
+          : sortString == "TotalDownloads"
+          ? 6
+          : sortString == "Category"
+          ? 7
+          : sortString == "GameVersion"
+          ? 8
+          : 1;
+    }
+
    private:
     inline auto getModSearchURL(SearchArgs& args) const -> QString override
     {
         auto gameVersionStr = args.versions.size() != 0 ? QString("gameVersion=%1").arg(args.versions.front().toString()) : QString();
-
+        
         return QString(
-                   "https://addons-ecs.forgesvc.net/api/v2/addon/search?"
+                   APPLICATION->getCFProxyURL()+"/v1/mods/search?"
```

Comment:
use QString place markers, i.e. `%1/v1/mods/search`

don't forget to change the place markers below as well